.github/workflows/gh-pages.yml

```diff
@@ -0,0 +1,65 @@
+name: Build
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout project
+        uses: actions/checkout@v3
+      - name: Checkout submodules
+        run: |
+          git submodule init
+          git submodule update
+      - name: Install build dependencies
+        run: |
+          sudo apt update
+          sudo apt install hugo
+      - name: Configure hugo
+        run: |
+          patch -o config.toml config.toml.example << EOF
+          1c1
+          < baseURL = "http://localhost/"
+          ---
+          > baseURL = "https://hugonikanor.github.io/r-selfhosted-wiki"
```

Comment:
If a pull request is coming into the main flow, this will need to be changed to something that ultimately reflects the end result of the landing page it will reside on.

Reply:
See commit 97c1205d18b7cd84cad52be22bcb0a4971d900be